rdmo/management/managers.py

```diff
@@ -0,0 +1,32 @@
+
+from django.db.models import QuerySet
+
+from rdmo.core.managers import (
+    AvailabilityQuerySetMixin,
+    ForCatalogQuerySetMixin,
+    ForGroupsQuerySetMixin,
+    ForSiteQuerySetMixin,
+)
+
+
+class ForProjectQuerySet(ForSiteQuerySetMixin, ForGroupsQuerySetMixin, ForCatalogQuerySetMixin,
```

Comment:
What do you think about this @jochenklar??
It cleans up a lot of code and is currently only used by `Views` and `Tasks`..